Switched badge house badge boost texts that were wrong/ Fix bug with list menu offset/ Fix bug in hall of fame

CreamElDudJafar opened the pull request:
- I know you fixed the gym text files, but they were still incorrect boosts for the Cerulean Badge House. Swapped them.

CreamElDudJafar committed:
Switched badge house badge boost texts that were wrong

CreamElDudJafar commented:
Oh okay. Just was changing it to match since Vermilion Gym says Defense now and Fuchsia says Speed now.

cRz-Shadows commented:
I'm second guessing myself gimme a bit, just checking with the team to see what it actually should be lol

CreamElDudJafar commented:
All good. Originally Thunderbadge said Speed and Soulbadge said Defense. But the texts were incorrect compared to the code. I think thats why you swapped them in a previous commit.

https://github.com/cRz-Shadows/Pokemon_Yellow_Legacy/commit/3c64464cee37a0ba05b1073866810e39c9e38d69

cRz-Shadows commented:
Yeah I think I'm just misremembering something. I thought we messed with badge boosts but it seems not outside of pikachu. I think you're right with this

CreamElDudJafar committed:
Fix Bug- List Menu Offsets

Could scroll past the bottom and the CANCEL button when on the 2nd or 1st row.

CreamElDudJafar commented:
One of your mods asked me to push commit to fix list menu scroll bug. It is in the PR now too.

CreamElDudJafar committed:
Fix Bug Where Player Would Take Half Step With Oak In Hall Of Fame

Does it with the new hide and show npcs added and the movement not clearing. Exists in vanilla but never is seen since its only 1 npc normally.

CreamElDudJafar committed:
Fix Part Of The Game Script Issue

From previous issue reported by someone else. But fixed so the predef jump doesnt fall through into the text script
